assets/scripts/segments/view.js

```diff
@@ -172,6 +172,35 @@ export function getVariantInfoDimensions (variantInfo, actualWidth = 0) {
   return { left: left / TILE_SIZE_ACTUAL, right: right / TILE_SIZE_ACTUAL, center: center / TILE_SIZE_ACTUAL }
 }
 
+const GROUND_LEVEL_OFFSETY = {
+  ASPHALT: 270, // offsetY = 11.25
+  CURB: 255.6, // offsetY = 10.65
+  RAISED_CURB: 196.8 // offsetY = 8.2
```

Comment:
It's so fun to change the numbers here and watch the sprites just move up and down / into place.

Some thoughts:

- Can we use integers here?
- Do we need to save the offsetY comments? Or can we remove them?
- I'm curious if we can redefine these numbers to be a little more intuitive. Right now lower elevations have a higher offsetY. Can we redefine them so that the order from low to high is inverted?

Reply:
- We can use integers here. I was trying to get as close to the original values as I could so I kept the decimals, but if we don't mind that they're slightly higher/lower than the original then I can round to the nearest integer.
- Nope, I had the offsetY comments to clarify how I got those values. I can remove them.
- Sorry, do you mean to redefine the values so `ASPHALT` has a lower value than `CURB` or to reorder the keys for `GROUND_LEVEL_OFFSETY` from low to high? If you meant to redefine the values from low to high, I can try but the math formula might have to be changed since lower elevations have a higher offsetY due to the ground having to be moved more downward.

Reply:
Yeah, I meant to redefined values so `ASPHALT` as a lower value than `CURB`. It's okay if the math formula would change as a result.